### ISSUE TITLE
Solar gains of horizontal window vary with azimuth

### ISSUE THREAD

**Issue by Linwal** [architecture-building-systems/RC_BuildingSimulator#40]
If I define a window with 
`alititude_tilt = 0`
the sum of the annual solar gains varies greatly with different azimuth angles
`azimuth_tilt=...`

I also get a problem that annual solar gains in Zurich for a south oriented window  (`azimuth_tilt=0`) reach their maximum with horizontal tilt (`alititude_tilt=0`) instead of somewhere around 35-45°.

Has this occurred  before?

For setting up my simulation I followed the Hourly Simulation of the Wiki.

**Comment by Linwal** [comment]
@pjayathissa 
I found the bug. The code is only made to work with vertical windows as the direct solar factor is set to 0 as long as the azimuths of the window and the sun position have an angle above 90°. This does not hold true as soon as tilt is involved. 

I propose replacing the following code in the _radiation.py_ script:
```
        if math.cos(sun_azimuth_rad - self.azimuth_tilt_rad) > 0:
            # Proportion of the radiation incident on the window (cos of the
            # incident ray)
            direct_factor = math.cos(sun_altitude_rad) * math.cos(sun_azimuth_rad - self.azimuth_tilt_rad) + \
                math.sin(sun_altitude_rad) * math.cos(self.alititude_tilt_rad)

        else:
            # If sun is behind the window surface
            direct_factor = 0
```
with:
```
   # Proportion of the radiation incident on the window (cos of the
        # incident ray)
        direct_factor = math.cos(sun_altitude_rad) * math.sin(self.alititude_tilt_rad) * math.cos(sun_azimuth_rad - self.azimuth_tilt_rad) + \
            math.sin(sun_altitude_rad) * math.cos(self.alititude_tilt_rad)


        # If the sun is in front of the window surface
        if(math.degrees(math.acos(direct_factor)) > 90):
            direct_factor=0

        else:
            pass

        return direct_factor
```

**Comment by pjayathissa** [comment]
Hey @Linwal, thanks for the find. Would you be able to make the change yourself and make a pull request?

Basically, you want to 
1) Branch the existing code
2) Make the changes
3) Push that branch up
4) In the github web-browser, make the pull request with the changes

I will then review the code and accept it

You can ask Daren if you need any help with this

**Comment by pjayathissa** [comment]
Fixed with Pull Request #41
